install/templates/argocd/application.yaml

```diff
@@ -19,6 +19,9 @@ spec:
       valueFiles:
       - "/values-global.yaml"
       - "/values-{{ .Values.main.clusterGroupName }}.yaml"
+      {{- if .Values.global.clusterVersion }}
```

Comment:
```suggestion
      # This assumes that the spokes share the same cluster version as the hub.
      # A limitation we are prepared to live with for now
      {{- if .Values.global.clusterVersion }}
```

Reply:
Not sure I follow here. This totally does work with hub and spoke having different ocp versions across the cluster. I tested with 4.10 on the hub and 4.9 on the spoke

Reply:
Ok added a note in Changes. (Not adding it in this template as I will drop it later on ;)